man/add_ws_to_parse_data.Rd

```diff
@@ -0,0 +1,28 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/parsed.R
+\name{add_ws_to_parse_data}
+\alias{add_ws_to_parse_data}
+\title{Pre-processing parse data}
+\usage{
+add_ws_to_parse_data(parse_data)
+}
+\arguments{
+\item{parse_data}{a parse table.}
+}
+\value{
+a pre-processed parse table.
+}
+\description{
+Pre-processing parse data
```

Comment:
I'd suggest we simply append commits and use regular push as a rule, and force-push only in special cases (e.g., cleanup of a long-living branch). This gives some free extra information about the evolution of the code.